### PR TITLE
Add 1.33 + EntitySchema extension

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: [T226152](https://phabricator.wikimedia.org/T226152)

### CODE REVIEW

**Comment by Ladsgroup** [comment]
You need to first build the image from docker files. Basically something like
```
docker build "wikibase/1.33/base" -t wikibase/wikibase:1.33 -t wikibase/wikibase:1.33-base -t wikibase/wikibase:latest -t wikibase/wikibase:latest-base
docker build "wikibase/1.33/bundle" -t wikibase/wikibase:1.33-bundle -t wikibase/wikibase:latest-bundle
```

Then doing compose up

**Comment by AlaaSarhan** [comment]
yeap that I tried as well and worked fine. I guess I confused docker-compose with docker-compose-build file and assumed this should be building .. all fine now

File: wikibase/1.33/bundle/Dockerfile [hunk 1]
```diff
@@ -11,14 +11,16 @@ bash download-extension.sh Elastica;\
...
 bash download-extension.sh cldr;\
-# Temporary, will remove once 1.34 is there
-curl -s "https://gerrit.wikimedia.org/r/plugins/gitiles/mediawiki/extensions/EntitySchema/+archive/99ec5bcaef2fdcc6ffa4ea9ca49b47c95ea76a0f.tar.gz" -o EntitySchema.tar.gz;\
 tar xzf OAuth.tar.gz;\
 tar xzf Elastica.tar.gz;\
 tar xzf CirrusSearch.tar.gz;\
 tar xzf UniversalLanguageSelector.tar.gz;\
 tar xzf cldr.tar.gz;\
 tar xzf WikibaseImport.tar.gz;\
+rm ./*.tar.gz
```
Comment (AlaaSarhan) [inline wikibase/1.33/bundle/Dockerfile]: this isn't needed as we are deleting everything on line 26, or?

Comment (Ladsgroup) [reply wikibase/1.33/bundle/Dockerfile]: In line 23 we are making a new tar.gz file again :D (EntitySchema.tar.gz) I can make it explicitly delete everything.

Comment (AlaaSarhan) [reply wikibase/1.33/bundle/Dockerfile]: I don't get it .. whatever tar.gz files that you make/download before line 26 will be deleted once you reach line 26 .. so line 20 seems redundant

**Pull request merged**